dev-python/deepdiff/deepdiff-6.3.0.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{9..11} )
+
+inherit distutils-r1 pypi
+
+DESCRIPTION="A library for comparing dictionaries, iterables, strings and other objects"
+HOMEPAGE="
+	https://github.com/seperman/deepdiff/
+	https://pypi.org/project/deepdiff/
+"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	>=dev-python/pyyaml-6.0[${PYTHON_USEDEP}]
+	>=dev-python/click-8.1.3[${PYTHON_USEDEP}]
+"
+
+DEPEND="
+	test? (
+		>=dev-python/jsonpickle-3.0.0[${PYTHON_USEDEP}]
+		>=dev-python/numpy-1.23.5[${PYTHON_USEDEP}]
+		>=dev-python/toml-0.10.2-r1[${PYTHON_USEDEP}]
```

Comment:
Ok, this is going to be a problem. We've mostly eliminated `dev-python/toml` now and it's not good to add more packages depending on it. Can you port it to use tomllib/tomli?

Reply:
the dump() method is being used in deepdiff and tomlib/tomli is parse only. Would have to patch load() and dump() separately.
How about new build for https://github.com/samuelcolvin/rtoml ? Using that is as simple as `import rtoml as toml`.
`tomlkit` is not as simple.

Reply:
The usual way to do it is to use `tomli-w` for dumping. `tomlkit` is already packaged.

Reply:
I don't know what to do with this.
tomlkit is not working out of the box. something fails a bit deeper. Maybe would ask upstream if they'd like to switch, it is slow? acording to tomli readme.
tomli/w would be an ugly multiline patch, file needs the file opened in binary mode, plus importerror checking.
I'd vote either for skipping tests or pulling in rtoml as it is based on *the* goto toml crate.

Reply:
That's really the worst idea possible. We already have to deal with 2 major TOML packages in Python, and what you're basically proposing is starting to point people towards a third one. Even worse, one that's not portable to all architectures supported by Gentoo.

Reply:
Upstream seems to be open to migration to tomli, so submitted PR https://github.com/seperman/deepdiff/pull/390